docs/content/docs/tutorials/ec2-example.md

```diff
@@ -0,0 +1,477 @@
+natgateways.ec2.services.k8s.aws             2022-10-15T01:58:27Z
+routetables.ec2.services.k8s.aws             2022-10-15T01:58:27Z
+securitygrouppolicies.vpcresources.k8s.aws   2022-09-30T23:00:35Z
+securitygroups.ec2.services.k8s.aws          2022-10-15T01:58:28Z
+subnets.ec2.services.k8s.aws                 2022-10-15T01:58:28Z
+transitgateways.ec2.services.k8s.aws         2022-10-15T01:58:28Z
+vpcendpoints.ec2.services.k8s.aws            2022-10-15T01:58:28Z
+vpcs.ec2.services.k8s.aws                    2022-10-15T01:58:28Z
+```
+ 
+* Check the **Custom Resource's** using `kubectl describe`:
+```
+kubectl describe vpcs
+kubectl describe internetgateways
+kubectl describe routetables
+kubectl describe natgateways
+kubectl describe elasticipaddresses
+kubectl describe subnets
+kubectl describe securitygroups
+```
+ 
+* Subnet gets into an 'available' state with a `ACK.ReferencesResolved = True` condition attached notifying users that the references (VPC, RouteTable) have been found and resolved:
+ 
+```
+Status:
+  Ack Resource Metadata:
+    Arn:                       arn:aws:ec2:us-west-2:<ID>:subnet/subnet-0ba22f5820bb41584
+    Owner Account ID:          <ID>
+    Region:                    us-west-2
+  Available IP Address Count:  4091
+  Conditions:
+    Last Transition Time:           2022-10-13T14:54:39Z
+    Status:                         True
+    Type:                           ACK.ReferencesResolved
+    Last Transition Time:           2022-10-13T14:54:41Z
+    Message:                        Resource synced successfully
+    Reason:
+    Status:                         True
+    Type:                           ACK.ResourceSynced
+  Default For AZ:                   false
+  Map Customer Owned IP On Launch:  false
+  Owner ID:                         515336597380
+  Private DNS Name Options On Launch:
+  State:      available
+  Subnet ID:  subnet-<ID>
+ 
+```
+ 
+### Validate
+ 
+This network setup should allow Instances deployed in the Private Subnet to connect to the internet. To validate this behavior deploy an Instance into the Private Subnet and the Public Subnet (bastion host). After deployments, `ssh` into the bastion host, then `ssh` into the Private Subnet Instance, and test internet connection. Security group is required by both instances launched in public and private subnets. 
+
+Note, we need to provide Subnet and SecurityGroup ID's in the yaml manually; run `kubectl describe subnets` and `kubectl describe securitygroups` commands to get ID's.
+ 
+* Deploy an Instance into the Private Subnet using provided yaml and `kubectl apply -f instance-private-subnet.yaml`:
+ 
+```
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Instance
+metadata:
+  name: tutorial-instance-private
+spec:
+  imageID: ami-02b92c281a4d3dc79 # AL2; us-west-2
+  instanceType: c3.large
+  subnetID: subnet-<private-ID>
+  securityGroupIDs:
+  - sg-<ID>
+  keyName: us-west-2-key # created via console
```

Comment:
Recommend we add a note in the text preceding this YAML that the user needs to create their own key via the AWS Console, ensure its file permissions are properly set, and supply the name of that key under `keyName` in both files.

Reply:
Already fixed in 361ded5b2363f99823f25feacb9b4c58fec6a5e7. 👍